recipes/simpleitk/bld.bat

```diff
@@ -0,0 +1,49 @@
+mkdir C:\b
+cd C:\b
+
+set "CMAKE_GENERATOR=Visual Studio 9 2008"
+if "%PY_VER%" == "3.3" set "CMAKE_GENERATOR=Visual Studio 10"
+if "%PY_VER%" == "3.4" set "CMAKE_GENERATOR=Visual Studio 10"
+if "%PY_VER%" == "3.5" set "CMAKE_GENERATOR=Visual Studio 14"
+
+if "%ARCH%" == "64" set "CMAKE_GENERATOR=%CMAKE_GENERATOR% Win64"
```

Comment:
Yes it does: [docs](http://conda.pydata.org/docs/building/environment-vars.html)